src/gt4py/backend/gtc_backend/numpy/backend.py

```diff
@@ -88,6 +88,8 @@ class GTCNumpyBackend(BaseBackend, CLIBackendMixin):
     name = "gtc:numpy"
     options: ClassVar[Dict[str, Any]] = {
         "oir_pipeline": {"versioning": True, "type": OirPipeline},
+        # TODO: Implement this option in source code
+        "ignore_np_errstate": {"versioning": True, "type": bool},
```

Comment:
I added the option `ignore_np_errstate` to the `gtc:numpy` backend (it didn't have it before), but I couldn't quickly figure out how to implement the feature cleanly.

Reply:
It was required to add some sort of backend option to keep `test_backend_option_order` in test_cli.py.

Reply:
I might forget something, but I think to implement this feature only two changes are needed:

1. Pass the option to `NpirCodegen.apply` inside `GTCNumpyBackend.generate_computation()`:
```python
ignore_np_errstate = self.builder.options.backend_opts.get("ignore_np_errstate" , True)
source = NpirCodegen.apply(self.npir, ignore_np_errstate=ignore_np_errstate)
```
2. Modify the Jinja template in `NpirCodegen.Computation` to wrap all the code inside `run()` with a context handler to modify the numpy settings if ignore_np_errstate is `True` or an empty handler to keep the current settings if is `False`. Something like this (actual Jinja syntax _might_ be slightly different) :
```python
            def run({{ signature }}):
            {% if ignore_np_errstate %}
                 with np.errstate(divide='ignore', over='ignore', under='ignore', invalid='ignore'):
            {% else %}
                 with np.errstate():
            {% endif %}
                     ....
```